contrib/zsh-completion/_packer

```diff
@@ -30,8 +30,12 @@ __inspect() {
 
 __push() {
   _arguments \
-    '-create[Create the build configuration if it does not exist].' \
-    '-token=[(<token>) Access token to use to upload.]'
+    '-m[(<detail>) A message to identify the purpose or changes in this Packer template.]' \
+    '-message=[(<detail>) A message to identify the purpose or changes in this Packer template.]' \
```

Comment:
`-m/message` is deprecated because it's not actually used. We'll remove this in 0.10